content/sensu-go/5.14/getting-started/learn-sensu.md

```diff
@@ -150,95 +145,95 @@ You should see a confirmation message from sensuctl.
 
 {{< highlight shell >}}
 sensuctl handler list
-{{< /highlight >}}
-
-You should see the `keepalive` handler.
-
-{{< highlight shell >}}
   Name      Type   Timeout   Filters   Mutator                                                   Execute                                                                                                              Environment Variables                            Assets         
 ─────────── ────── ───────── ───────── ───────── ────────────────────────────────────────────────────────────────────────────────────────────────────────── ────────────────────────────────────────────────────────────────────────────────────────────────── ───────────────────── 
- keepalive   pipe         0                       RUN:  /usr/local/bin/sensu-slack-handler -c "${KEEPALIVE_SLACK_CHANNEL}" -w "${KEEPALIVE_SLACK_WEBHOOK}"   KEEPALIVE_SLACK_WEBHOOK=https://hooks.slack.com/services/XXX,KEEPALIVE_SLACK_CHANNEL=#monitoring   sensu-slack-handler  
+ keepalive   pipe         0                       RUN:  /usr/local/bin/sensu-slack-handler -c "${KEEPALIVE_SLACK_CHANNEL}" -w "${KEEPALIVE_SLACK_WEBHOOK}"   KEEPALIVE_SLACK_WEBHOOK=https://hooks.slack.com/services/ABC,KEEPALIVE_SLACK_CHANNEL=#monitoring   sensu-slack-handler  
 {{< /highlight >}}
 
-You should now see monitoring events in Slack indicating that the sandbox entity is in an OK state.
+Sensu monitoring events should begin arriving in your Slack channel, indicating that the sandbox entity is in an OK state.
 
 **4. Filter keepalive events**
 
-Now that we're generating Slack alerts, let's reduce the potential for alert fatigue by adding a filter that only sends only warning, critical, and resolution alerts to Slack.
+Now that you're generating Slack alerts, let's reduce the potential for alert fatigue by adding a filter that sends only warning, critical, and resolution alerts to Slack.
 
-To accomplish this, we'll interactively add the built-in is_incident filter to the keepalive handler so we'll only receive alerts when the sandbox entity fails to send a keepalive event.
+To accomplish this, you'll interactively add the built-in is_incident filter to the keepalive handler, which will make sure you only receive alerts when the sandbox entity fails to send a keepalive event.
 
 {{< highlight shell >}}
 sensuctl handler update keepalive
 {{< /highlight >}}
 
-When prompted for the filters selection, enter `is_incident` to apply the incidents filter.
+The first prompt will be for environment variables. Just press `return` to continue. The second prompt is for the filters selection--enter `is_incident` to apply the is_incident filter.
 
 {{< highlight shell >}}
-? Filters: [? for help] is_incident
+? Filters: is_incident
 {{< /highlight >}}
 
-We can confirm that the keepalive handler now includes the incidents filter using sensuctl:
+For the mutator, timeout, type, runtime assets, and command prompts, just press `return` again.
```

Comment:
```suggestion
For the mutator, timeout, type, runtime assets, and command prompts, just press `return` for each of them.
```

Reply:
Or maybe `for each prompt`?

Reply:
Yeah, I don't like "for each of them" either now that I re-read this.

Reply:
That works!